src/main/java/in/cognitron/lms/service/Hello.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 cognitron.in
+ * all rights reserved
+ */
+
+package in.cognitron.lms.service;
+
+import org.simplity.fm.core.Message;
+import org.simplity.fm.core.MessageType;
+import org.simplity.fm.core.serialize.IInputObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import in.cognitron.lms.LmsService;
+import in.cognitron.lms.LmsServiceContext;
+
+public class Hello extends LmsService {
+	protected static final Logger logger = LoggerFactory.getLogger(Hello.class);
+	private static final String SERVICE_NAME = "Hello";
+	@Override
+	public String getId() {
+		return SERVICE_NAME;
+	}
```

Comment:
Linting nitpicks:
1. I would recommend enabling four-space tabs for indents instead of `\t`.
2. Your linter doesn’t skip a line (or doesn’t ask you to skip a line) before the implementation of a function?  For example:
```
protected static final Logger logger = LoggerFactory.getLogger(Hello.class);
private static final String SERVICE_NAME = "Hello";

@Override
public String getId() {
    return SERVICE_NAME;
}
```
would be easier to read than a wall of text:
```
protected static final Logger logger = LoggerFactory.getLogger(Hello.class);
private static final String SERVICE_NAME = "Hello";
@Override
public String getId() {
    return SERVICE_NAME;
}
```